global.json

```diff
@@ -1,3 +1,6 @@
 {
-    "projects": ["src", "test/WebSites", "samples"]
+    "projects": [ "src", "test/WebSites", "samples" ],
+    "sdk": {
+        "version": "1.0.0-preview2-003121"
```

Comment:
We need them to be in sync because KoreBuild is the one that ensures the SDK is installed. I can start a change to KoreBuild now to do that first.